modules/atomic-widgets/assets/js/editor/commands/styles.js

```diff
@@ -1,3 +1,5 @@
+import { getVariantByMeta } from '../utils/get-variants';
```

Comment:
Make sure to remove the StyleDef and Variant if no need

Reply:
Done - added upon apply, checking first if no values other than `undefined` - if none it will remove the variant.

Then it takes the current style from the model again (as we're working with a structuralClone object), checks for the length of variants and if its 0 - will call for the internal command to delete the style